src/Tribe/Tickets_Handler.php

```diff
@@ -789,4 +792,15 @@ public function get_attendee_report_link( $post ) {
 		return $url;
 	}
 
+	/**
+	 * Sets the `$typenow` var to the post_type
+	 *
+	 * Causes parent post type to be highlighted in admin menus.
+	 *
+	 * @see `admin_head` action
+	 */
+	public function set_typenow() {
+		global $typenow;
+		$typenow = $_REQUEST['post_type'];
```

Comment:
Can you bail out of this function if `empty( $_REQUEST['post_type'] )`?

Reply:
Sure. That will only happen if it's not attached to `admin_head`, but just in case I can add that check.